File: FoodDelivery_Angular_Frontend/Frontend-App/src/app/Models/customerInfo.ts
export class custInfoModel {
    cust_id: number;
    cust_type_cd: string;
    cust_name: string;
    cust_email: string;
    cust_phno: number;
    cust_pin: number

    constructor() {
        this.cust_id = 0;
        this.cust_type_cd = '';
        this.cust_name = '';
        this.cust_email = '';
        this.cust_phno = 0;
        this.cust_pin = 0;
    }

}

File: FoodDelivery_Backend/Models/CustomerInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FoodDelivery_Backend.Models
{
    public class CustomerInfo
    {
        public decimal cust_id { get; set; }
        public string cust_type_cd { get; set; }
        public string cust_name { get; set; }
        public string cust_email { get; set; }
        public decimal cust_phno { get; set; }
        public decimal cust_pin { get; set; }
        public string cust_passwd { get; set; }

        public CustomerType custType { get; set; }

    }
}